apps/api/schedule.py

```diff
@@ -169,6 +170,69 @@ def post(self, proposal_type):
         return [t.id for t in res]
 
 
+class ProposalC3VOCPublishingWebhook(Resource):
+    method_decorators = {"post": [_require_video_api_key]}
+
+    def post(self):
+        if not request.is_json:
+            abort(415)
+
+        payload = request.get_json()
+
+        try:
+            conference = payload["fahrplan"]["conference"]
+            proposal_id = payload["fahrplan"]["id"]
+        except KeyError:
+            abort(422)
+
+        if not payload["is_master"]:
+            # c3voc *should* only send us information about the master
+            # encoding getting published. Aborting early ensures we don't
+            # accidentially delete video information from the database.
+            abort(403)
+
+        if conference != f"emf{event_year()}":
+            abort(422)
+
+        proposal = Proposal.query.get_or_404(proposal_id)
+
+        if payload["voctoweb"]["enabled"]:
+            if payload["voctoweb"]["frontend_url"]:
+                proposal.c3voc_url = payload["voctoweb"]["frontend_url"]
```

Comment:
```suggestion
        if payload.get("voctoweb", {}).get("enabled") is True:
            if c3voc_url := payload["voctoweb"].get("frontend_url"):
                proposal.c3voc_url = c3voc_url
```

Reply:
I highly despise the `:=`-Syntax recent python versions introduced, because (in my opinion) they make the code less readable. I have implemented parts of that suggestion anyway.